todo_app/app.py

```diff
@@ -1,48 +1,40 @@
-import json
-
 from flask import Flask, redirect, render_template, request
 
-from todo_app.data.exceptions import ResponseError
-from todo_app.data.trello_items import Status, TrelloRequests
+from todo_app.data.items import Status
+from todo_app.data.mongodb_requests import MongoDbRequests
 from todo_app.flask_config import Config
 
 
 def create_app() -> Flask:
     app = Flask(__name__)
     app.config.from_object(Config())
 
-    trello_requests = TrelloRequests()
-
-    trello_requests.init_lists()
+    mongodb_reqeusts = MongoDbRequests()
```

Comment:
Small but a typo that then gets repeated quite a bit!
```suggestion
    mongodb_requests = MongoDbRequests()
```